README.md

```diff
@@ -238,6 +238,72 @@ fga store **import**
 {}
 ```
 
+##### Export Store
+
+###### Command
+fga store **export**
+
+###### Parameters
+* `--store-id`: Specifies the store to export
+* `--output-file`: The file to output the store to (optional, writes to the terminal if omitted)
+* `--model-id`: Specifies the model to export (optional, exports the latest model if omitted)
+* `--max-tuples`: Specifies the max number of tuples to include in the output (option, defaults to 100)
+
+###### Example
+`fga store export`
```

Comment:
Sorry I missed this earlier, as we require a store ID I think it should be in the example command
```suggestion
`fga store export --store-id=01H0H015178Y2V4CX10C2KGHF4`
```